.gitignore

```diff
@@ -14,3 +14,13 @@
 # System Files
 .DS_Store
 Thumbs.db
+# Terraform secrets
+*.tfvars
+.terraform
+.terraform.lock.hcl
+*.tfstate
+*.tfstate.backup
+**/.terraform
+# Editor files
+.vscode
+.idea
```

Comment:
Theese should probably not be ignored, .vscode has required extensions for formatting at least.

Reply:
Personally I'm of the opinion that the local dev environment should be up the the person working with it so it's my general approach to remove editor specific stuff.